scripts/run_cyclictest.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+import argparse
+
+parser = argparse.ArgumentParser(description="Run cyclictest command.")
+parser.add_argument('output_file', type=str, help='File to output the result.')
```

Comment:
If these arg is not provided, this will probably fail. Add required=true

Reply:
I created a positional argument (no -- option in front of the name). Positional arguments are always required so I cannot use required=True for such options